src/vellum/workflows/nodes/displayable/inline_prompt_node/node.py

```diff
@@ -46,18 +46,38 @@ def run(self) -> Iterator[BaseOutput]:
             )
 
         string_outputs = []
+        json_outputs = []
+
+        should_parse_json = False
+        if hasattr(self, "parameters"):
+            custom_params = getattr(self.parameters, "custom_parameters")
```

Comment:
[nit] should be able to just do `self.parameters.custom_parameters` without issue